### DISCUSSION TITLE
Image Links Broken

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#12316] [Help]
I logged in today to do some work and now random image links are broken. Nothing has been changed since last login as far as design or database. 

<img width="1310" alt="image" src="https://github.com/Budibase/budibase/assets/99847728/f2e8b6dc-fd27-4f99-bd1a-b79a3495f602">

This is super problematic as I have specification sheets that reference these images and are public viewable... see example below. Images don't show up anymore. 

**https://ciaoimports.budibase.app/app/portal#/spec-sheet/standard/ZAC476**

**Comment by stellarfinn** [comment]
@melohagan @mjashanks @joebudi 

I love Budibase, but these intermittent problems are becoming exhaustive. We're paying for Budibase Cloud, can someone please explain why, without making any changes ourselves, images suddenly disappear. This is an active app that is used in every day commerce. I cannot justify Budibase, or upgrading to a higher tier, to my boss any longer if this continues.

**Reply by melohagan** [reply]
Hey @stellarfinn Sorry to hear you're having issues; were these images uploaded to attachment fields in an internal DB?

**Reply by stellarfinn** [reply]
Hi, yes they were. Please see attached screenshot of uploaded photos in database. It appears to be random image loss. 


<img width="1909" alt="image" src="https://github.com/Budibase/budibase/assets/99847728/a53b5405-d818-44d8-a2a6-5a8dfc77bfc2">

Brands missing images on app. 

<img width="1899" alt="image" src="https://github.com/Budibase/budibase/assets/99847728/da5d4624-0f8d-450d-bed7-63341de6e3a0">

Brands with broken image links in database. 

<img width="1892" alt="image" src="https://github.com/Budibase/budibase/assets/99847728/3723f5a0-92b0-48b3-b105-331bf408a212">

**Reply by melohagan** [reply]
Just out of curiosity, could you sort the rows by the *Updated At* time column if available, to see if there's a pattern there.

**Reply by melohagan** [reply]
Also sort by Auto ID please and share the screenshots

**Reply by stellarfinn** [reply]
@melohagan 

Sorted by Updated At
<img width="1255" alt="image" src="https://github.com/Budibase/budibase/assets/99847728/246066fe-a6c4-4aac-8208-67dbc96d2ac9">


Sorted by Auto ID
<img width="1147" alt="image" src="https://github.com/Budibase/budibase/assets/99847728/180cc7a6-6bfe-40f9-a085-c2b4a9c744d0">

**Reply by stellarfinn** [reply]
@melohagan It appears this is related to the failed backup from my previous discussion post. 

I previously went through and uploaded all missing photos when I recovered my app from a backup. Somehow all my updates disappeared and now I'll have to reupload them again.

**Reply by melohagan** [reply]
> It appears this is related to the failed backup from my previous discussion post.

Could you share a link to that discussion please.
